### PR TITLE
🎉 preserve scrollY on gdoc refresh

### PR DESCRIPTION
**Pull request by ikesau**
Just a little QOL thing

https://github.com/owid/owid-grapher/assets/11844404/3b3d11a4-2176-4b1f-ac1f-091ecb7fb1ea

### CODE REVIEW

File: adminSiteClient/GdocsPreviewPage.tsx [hunk 1]
```diff
@@ -115,13 +118,23 @@ export const GdocsPreviewPage = ({ match, history }: GdocsMatchProps) => {
...
             setCurrentGdoc({
                 ...latestGdoc,
                 slug: currentGdoc.slug,
                 published: currentGdoc.published,
                 publishedAt: currentGdoc.publishedAt,
                 publicationContext: currentGdoc.publicationContext,
             })
+            // Should only run this if the iframe has reloaded
+            setTimeout(() => {
+                if (latestGdoc.revisionId != revisionId) {
```
Comment (marcelgerber) [inline adminSiteClient/GdocsPreviewPage.tsx:132]: I don't see where `revisionId` is updated?

Comment (ikesau) [reply adminSiteClient/GdocsPreviewPage.tsx:132]: It's part of the spread in `latestGdoc`

It's given to us by Google and it only changes if the Google Doc itself changes, and is also used as the `key` on the iframe below, which is what triggers React to re-render it there, too.

**Comment by mlbrgl** [comment]
Current dependencies on/for this PR:
* master
  * **PR #2328** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/2328" target="_blank">Graphite</a>  👈
    * **PR #2340** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/2340" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/owid/owid-grapher/2328?utm_source=stack-comment).

**Review by mlbrgl — COMMENTED** [review]
Thank you! 

I'm getting the desired behaviour on FF but not on Chrome. Do you mind taking a look at #2340, where I suggested a fix?

**Review by mlbrgl — APPROVED** [review]

**Pull request merged**